docs/modules/ROOT/partials/cmos-health-checks.adoc

```diff
@@ -47,6 +47,17 @@ In virtual environments, high CPU steal indicates that the virtual machines migh
 
 *Further Reading*: https://docs.couchbase.com/server/current/learn/buckets-memory-and-storage/storage.html[Storage]
 
+[#CB90044]
+=== Babysitter Managed Process Crash (CB90043)
+
+*Background*: Babysitter is part of Couchbase Server's cluster manager which is responsible for maintaining a variety of Couchbase Server processes.
```

Comment:
I'd perhaps include more about what it does - for example, "it restarts processes that exit for whatever reason" (or words to that effect, I'm not a technical writer :p )